lib/fern/api.rb

```diff
@@ -8,14 +8,43 @@ module Api
 
     included do
       class_eval { class_attribute :fern }
-      self.fern = {}
-      def present(obj)
-        render json: obj
+
+      def form
+        if @form.nil?
+          p = params
```

Comment:
Do we want to support strong params here, I wonder? Cerulean included the following:
```ruby
p = p.permit! if params.respond_to?(:permit!)
```